src/content/docs/pt-br/guides/images.mdx

```diff
@@ -664,13 +708,20 @@ export default defineConfig({
 });
 ```
 
-:::note
-Ao usar um [gerenciador de pacotes rigoroso](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) como o `pnpm`, talvez seja necessário instalar o Sharp manualmente o seu projeto, mesmo que seja uma dependência do Astro:
+### Configurando um serviço sem operação
 
-```bash
-pnpm install sharp
+Se o seu [adaptador para modo `servidor` ou `híbrido`](https://astro.build/integrations/?search=&categories%5B%5D=adapters) não suporta a otimização incluída no Astro  com Squoosh e Sharp (e.x. Deno, Cloudflare), você pode configurar um serviço de imagem sem operação para permitir que você use os componentes `<Image />` e `<Picture />`. Note que o Astro não faz nenhuma transformação de imagem ou processamente nesse ambiente. No entanto, você ainda pode aproveitar os outros benefícios de usar `astro:assets`, incluindo a prevenção de Cumulative Layout Shift (CLS), a imposição do atributo `alt`, e uma experiência consistente no uso.
```

Comment:
Since these are API options, they shouldn't be translated, despite making the sentence structure a bit harder to read. 
```suggestion
Se o seu [adaptador para modo `server` ou `hybrid`](https://astro.build/integrations/?search=&categories%5B%5D=adapters) não suporta a otimização incluída no Astro com Squoosh e Sharp (e.x. Deno, Cloudflare), você pode configurar um serviço de imagem sem operação para permitir que você use os componentes `<Image />` e `<Picture />`. Note que o Astro não faz nenhuma transformação de imagem ou processamente nesse ambiente. No entanto, você ainda pode aproveitar os outros benefícios de usar `astro:assets`, incluindo a prevenção de Cumulative Layout Shift (CLS), a imposição do atributo `alt`, e uma experiência consistente no uso.
```